datm/cime_config/stream_definition_datm.xml

```diff
@@ -778,12 +849,12 @@
 
   <!-- Generic single point site  -->
 
-  <stream_entry name="CLM_USRDAT.$CLM_USRDAT_NAME">
+  <stream_entry name="CLM_USRDAT.NEON">
     <stream_meshfile>
       <meshfile>none</meshfile>
     </stream_meshfile>
     <stream_datafiles>
-      <file first_year="$DATM_YR_START" last_year="$DATM_YR_END">$DIN_LOC_ROOT_CLMFORC/$CLM_USRDAT_NAME/CLM1PT_data/%ym.nc</file>
+      <file first_year="$DATM_YR_START" last_year="$DATM_YR_END">$DIN_LOC_ROOT_CLMFORC/NEON/CLM1PT_data/%ym.nc</file>
```

Comment:
This will also need some adjustments (probably in buildnml) to not be hardcoded.